src/helpers/projects.ts

```diff
@@ -0,0 +1,16 @@
+import { ICategory, IConvertedCategories } from '@/apollo/types/types';
```

Comment:
since this file is not related to projects page, I think you can rename it to 'singleProject.ts'